scripts/install

```diff
@@ -7,175 +7,95 @@
-###		- And the section "SETUP LOGROTATE" in the upgrade script
-
-# Use logrotate to manage application logfile(s)
-ynh_use_logrotate
-
-# Create a dedicated Fail2Ban config
-ynh_add_fail2ban_config --logpath="/var/log/nginx/${domain}-error.log" --failregex="Regex to match into the log for a failed login"
-
 #=================================================
 # APP INITIAL CONFIGURATION
 #=================================================
 # ADD A CONFIGURATION
 #=================================================
 ynh_script_progression --message="Adding a configuration file..." --weight=1
 
-### You can add specific configuration files.
-###
-### Typically, put your template conf file in ../conf/your_config_file
-### The template may contain strings such as __FOO__ or __FOO_BAR__,
-### which will automatically be replaced by the values of $foo and $foo_bar
-###
-### ynh_add_config will also keep track of the config file's checksum,
-### which later during upgrade may allow to automatically backup the config file
-### if it's found that the file was manually modified
-###
-### Check the documentation of `ynh_add_config` for more info.
-
-# Install Node.JS
-ynh_script_progression --message="Installing Node.JS..." --weight=1
-ynh_install_nodejs --nodejs_version="$nodejsversion"
-
-#Install Yarn
-ynh_script_progression --message="Installing Yarn..." --weight=1
-ynh_install_extra_app_dependencies --repo="deb https://dl.yarnpkg.com/debian/ stable main" --package="yarn" --key="https://dl.yarnpkg.com/debian/pubkey.gpg"
-
-# Install Symfony
-ynh_script_progression --message="Installing Symfony..." --weight=1
-ynh_install_extra_app_dependencies --repo="deb https://dl.cloudsmith.io/public/symfony/stable/deb/debian stable main" --package="symfony_cli" --key="https://dl.cloudsmith.io/public/symfony/stable/gpg.99CEF8A62216B73D.key"
-
 # Configure .env
 ynh_add_config --template=".env" --destination="$install_dir/.env"
 
-ynh_exec_warn_less ynh_install_composer --phpversion="$phpversion" --workdir="$install_dir"
-ynh_exec_warn_less ynh_composer_exec --phpversion="$phpversion" --workdir="$install_dir" --commands="dump-env prod"
-ynh_exec_warn_less ynh_composer_exec --phpversion="$phpversion" --workdir="$install_dir" --commands="install --prefer-dist --no-dev --no-autoloader --no-scripts --no-progress"
-ynh_exec_warn_less ynh_composer_exec --phpversion="$phpversion" --workdir="$install_dir" --commands="clear-cache"
+chmod 400 "$install_dir/.env"
+chown $app:$app "$install_dir/.env"
+
+#=================================================
+# INSTALL KBIN WITH COMPOSER
+#=================================================
+ynh_script_progression --message="Installing $app with Composer..." --weight=5
+
+#ynh_install_composer_as_user --phpversion=$phpversion --install_args="--prefer-dist --no-autoloader --no-scripts --no-progress"
+#ynh_install_composer_as_user --phpversion=$phpversion --install_args="dump-env prod"
+ynh_install_composer_asuser --phpversion=$phpversion
```

Comment:
I'm testing this for fun, in case you're interested, the install failed here at ynh_install_composer_asuser --phpversion=$phpversion with "command not found", I notice that the lines above have the command spelled with an underscore like "...as_user", perhaps that's the reason?  I'll try to figure out a way to modify that line and run the installer again.  Thanks for your efforts so far.

Reply:
Hey you're right about that one! It had somehow slipped out of my eyes, gonna try correcting that line at home